crates/primitives-traits/Cargo.toml

```diff
@@ -92,4 +92,5 @@ serde = [
     "revm-primitives/serde",
     "roaring/serde",
 	"revm-primitives/serde",
-]
\ No newline at end of file
+]
+reth-codec = []
```

Comment:
why is this a feature if reth-codecs dep is non optional?

Reply:
will make the dep optional. unless the `reth-codec` feature is enabled for `reth-primitives`, none of the `reth-primitives` types will impl their corresponding `Full`- trait.